documentation/api/io.strimzi.api.kafka.model.KafkaAuthorizationSimple.adoc

```diff
@@ -0,0 +1,27 @@
+Simple authorization in Strimzi uses the `SimpleAclAuthorizer` plugin, the default Access Control Lists (ACLs) authorization plugin provided with Apache Kafka.
```

Comment:
I'm afraid this changed since this Pr was created.

```suggestion
Simple authorization in Strimzi uses the `AclAuthorizer` plugin, the default Access Control Lists (ACLs) authorization plugin provided with Apache Kafka.
```

Reply:
Changed all references in the doc files